src/sqlfluff/core/rules/std/L039.py

```diff
@@ -32,6 +32,7 @@ def _eval(self, segment, parent_stack, **kwargs):
         # For the given segment, lint whitespace directly within it.
         prev_newline = True
         prev_whitespace = None
+        violations = []
```

Comment:
I don't see any corresponding changes to the test file, `L039.yml`. Should there be new or updated tests to go with this?

Reply:
Nope. Existing tests are fine. The new changes will just make it faster.